app/src/main/res/values-ar/strings.xml

```diff
@@ -453,4 +453,5 @@
   <string name="faq_answer_7">&lt;p&gt;إذا لم يتم تشغيل الصوت الخاص بك&lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;\n&lt;p&gt;تحقق لمعرفة ما إذا كان التطبيق محدثًا أم لا:&lt;/p&gt;\n&lt;p&gt; &lt;li&gt;انتقل إلى متجر Play وتأكد من تحديث التطبيق إلى أحدث إصدار&lt;/li&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;\n&lt;p&gt;تحقق من اتصالك بالإنترنت:&lt;/p&gt;&lt;p&gt; &lt;li&gt;إذا كان اتصالك بالإنترنت بطيئًا ، فحاول إعادة الاتصال بشبكة Wi-Fi أو الاتصال بشبكة أخرى. قد يتسبب الإنترنت البطيء في تحميل الصوت بشكل غير منتظم ، مما يجعل من الصعب تشغيله.&lt;/li&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;\n&lt;p&gt;اطلب من المسؤول التحقق من أجهزتهم واتصال الإنترنت:&lt;/p&gt;&lt;p&gt; \n&lt;li&gt;اطلب من المسؤول استكشاف الأخطاء وإصلاحها باستخدام الخطوات المذكورة أعلاه&lt;/li&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;\n&lt;p&gt;أخبرنا إذا كنت لا تزال تواجه مشكلات في التحميل:&lt;/p&gt;&lt;p&gt; &lt;li&gt;أبلغ عن مشكلة عن طريق الاتصال بنا على admin@oppia.org.&lt;/li&gt;&lt;/p&gt;</string>
   <string name="faq_answer_8">&lt;p&gt;لتنزيل استكشاف:&lt;/p&gt;\n&lt;p&gt;1. من الصفحة الرئيسية ، انقر فوق موضوع أو استكشاف.&lt;/p&gt;\n&lt;p&gt;2. من صفحة الموضوع هذه ، انقر على علامة التبويب &lt;strong&gt; معلومات &lt;/ strong&gt;.&lt;/p&gt;\n&lt;p&gt;3. اضغط على &lt;strong&gt; تنزيل الموضوع &lt;/ strong&gt;.&lt;/p&gt;\n&lt;p&gt;4. اعتمادًا على إعدادات التطبيق ، قد تحتاج إلى موافقة المسؤول أو اتصال Wifi ثابتًا لإكمال التنزيل. إذا لزم الأمر ، بمجرد استيفاء هذه المتطلبات ، يتم تنزيل الموضوع على الجهاز ويمكن استخدامه في وضع عدم الاتصال بواسطة جميع ملفات التعريف. &lt;p&gt;</string>
   <string name="faq_answer_9">&lt;p&gt; إذا لم تتمكن من العثور على سؤالك أو كنت ترغب في الإبلاغ عن خطأ ، فاتصل بنا على admin@oppia.org. &lt;/p&gt;</string>
+  <string name="profile_edit_activity_title">تعديل الملف الشخصي</string>
```

Comment:
@shankarpriyank usually we don't add this from our side, but we can take suggestions from @rt4914.

Reply:
Actually in the last pr which I messed up I was asked to add the string in arabic too

Reply:
@BenHenning Is this correct? We do add strings for other languages too, right?

Reply:
We should rename strings when we can, but new strings are added through Translatewiki since we have volunteer translators specifically to help with that task.

Reply:
@rt4914 Should I undo my changes in the xml file for the Arabic and Portugal language?

Reply:
Please do @shankarpriyank. They'll be translated after this PR is merged by contributors on Translatewiki.

Reply:
Made the changes as requested , PTAL @rt4914 @BenHenning